cmd/kubeseal/main.go

```diff
@@ -211,6 +215,11 @@ func main() {
 	flag.Parse()
 	goflag.CommandLine.Parse([]string{})
 
+	if *printVersion {
+		fmt.Printf("%s\n", VERSION)
```

Comment:
`fmt.Printf("kubeseal version: %s\n", VERSION)`

Just so that the output "scales" sensibly when we want to report the versions of other components (client-go?) too one day.

Reply:
Yeah, I guess let's assume it's just for display for now rather than automated use.